server/api/sample_sizes.py

```diff
@@ -20,11 +20,14 @@ def sample_size_options(
 ) -> Dict[str, Dict[str, ballot_polling.SampleSizeOption]]:
     if not election.contests:
         raise BadRequest("Cannot compute sample sizes until contests are set")
-    if not election.risk_limit:
+    if election.risk_limit is None:
         raise BadRequest("Cannot compute sample sizes until risk limit is set")
-    risk_limit = float(election.risk_limit) / 100
 
     def sample_sizes_for_contest(contest: Contest):
+        # For some reason putting this here works, but not if it's in the outer
+        # function
+        if election.risk_limit is None:
+            raise BadRequest("Cannot compute sample sizes until risk limit is set")
```

Comment:
maybe just `assert election.risk_limit is not None` then, since we know it's ok at this point